flottform/demo/src/routes/custom-ui/+page.svelte

```diff
@@ -0,0 +1,586 @@
+					<label for="name">Name</label>
+					<input
+						class="border-2 box-border border-primary-blue rounded-md py-2 px-4"
+						id="name"
+						type="text"
+						name="name"
+						bind:value={$prefilledForm.name}
+					/>
+				</div>
+				<div class="flex flex-col">
+					<label for="surname">Surname</label>
+					<input
+						class="border-2 box-border border-primary-blue rounded-md py-2 px-4"
+						id="surname"
+						type="text"
+						name="surname"
+						bind:value={$prefilledForm.surname}
+					/>
+				</div>
+			</div>
+			<div class="grid grid-cols-1 sm:grid-cols-2 gap-6">
+				<div class="flex flex-col">
+					<label for="email">E-Mail</label>
+					<input
+						class="border-2 box-border border-primary-blue rounded-md py-2 px-4"
+						id="email"
+						type="email"
+						name="email"
+						bind:value={$prefilledForm.email}
+					/>
+				</div>
+				<div class="flex flex-col">
+					<label for="phone">Phone number</label>
+					<input
+						class="border-2 box-border border-primary-blue rounded-md py-2 px-4"
+						id="phone"
+						type="tel"
+						name="phone"
+						bind:value={$prefilledForm.phone}
+					/>
+				</div>
+			</div>
+			<div class="grid grid-cols-1 sm:grid-cols-3 gap-6">
+				<div class="flex flex-col sm:col-span-2">
+					<label for="street">Street</label>
+					<input
+						class="border-2 box-border border-primary-blue rounded-md py-2 px-4"
+						id="street"
+						type="text"
+						name="street"
+						bind:value={$prefilledForm.street}
+					/>
+				</div>
+				<div class="flex flex-col">
+					<label for="houseNumber">House number</label>
+					<input
+						class="border-2 box-border border-primary-blue rounded-md py-2 px-4"
+						id="houseNumber"
+						type="text"
+						name="houseNumber"
+						bind:value={$prefilledForm.houseNumber}
+					/>
+				</div>
+				<div class="flex flex-col sm:col-span-2">
+					<label for="city">City</label>
+					<input
+						class="border-2 box-border border-primary-blue rounded-md py-2 px-4"
+						id="city"
+						type="text"
+						name="city"
+						bind:value={$prefilledForm.city}
+					/>
+				</div>
+				<div class="flex flex-col">
+					<label for="postcode">Postcode</label>
+					<input
+						class="border-2 box-border border-primary-blue rounded-md py-2 px-4"
+						id="postcode"
+						type="text"
+						name="postcode"
+						bind:value={$prefilledForm.postcode}
+					/>
+				</div>
+			</div>
+			<div class="grid grid-cols-1">
+				<div class="flex flex-col">
+					<label for="problemDescription">Please describe your problem</label>
+					<textarea
+						class="border-2 box-border border-primary-blue rounded-md py-2 px-4"
+						id="problemDescription"
+						name="problemDescription"
+						rows="4"
+						bind:value={$prefilledForm.problemDescription}
+					/>
+				</div>
+			</div>
+			<div class="grid gap-6 grid-cols-1 sm:grid-cols-[390px_1fr]">
+				<FileInput id="document" name="document" {fileInput} />
+
```

Comment:
Here we need to add our Flottform button 
```suggestion
				<button
					class="border rounded p-2 h-fit self-center border-primary-blue"
					bind:this={createChannelButton}
					type="button">Flottform</button
				>
```